### ISSUE TITLE
[Bug Report]: Crash on startup on 5.0.10-1.19

### ISSUE THREAD

**Issue by BlueCorallite** [chronosacaria/MCDungeonsWeapons#217]
### Describe the bug; What happened?

Hi, I'm trying to use the MCDungeonsWeapons mod for 1.19.2 along with a bunch of mods but the game crashes on startup when trying to do so. I checked every mod I'm using and I narrowed it down to DungeonsWeapons as the one that causes the crash. Other mods like DungeonsArtifacts and DungeonsArmors work fine, so I'm not sure why this one is causing problems.

Here is the log if it helps. Any help would be appreciated.
[latest.log](https://github.com/chronosacaria/MCDungeonsWeapons/files/11492570/latest.log)

### Expected Behavior

Not crashing and to launch properly, I don't know what else to say.

### Mod Version

5.0.10-1.19

### Minecraft Launcher

Vanilla

### Operating System

Windows

**Comment by chronosacaria** [comment]
🤔 Not sure why this is still happening. Thanks for reporting this. I'll look into it!

**Comment by BlueCorallite** [comment]
Of course!

**Comment by chronosacaria** [comment]
Actually, @BlueCorallite, it looks like MCDW isn't causing an issue; at least, not one that is showing up on the log. Does the crash happen without MCDW installed? I know you said you narrowed it down the MCDW, but I'm not seeing a place where it is explicitly involved with the crash

**Comment by BlueCorallite** [comment]
The game launches fine without it. When I put it in the mods folder and try again, it crashes.

**Comment by chronosacaria** [comment]
Actually, as a matter of fact, MCDW isn't loaded in that log, at all:
```
- mcda 2.4.4
- mcdar 1.5.3-1.19
[MCDW would be right here if it was loaded]
- mcsa 3.1.4+1.19
```

**Comment by chronosacaria** [comment]
Ah, okay. Could you add it and post that crash, please? That way I can see what the issue is in relation to MCDW

**Comment by BlueCorallite** [comment]
oh. lmao.
Let me try again

**Comment by BlueCorallite** [comment]
[latest.log](https://github.com/chronosacaria/MCDungeonsWeapons/files/11492648/latest.log)
This should have it.

**Comment by chronosacaria** [comment]
Okay, that's the same issue that Daedelus had. I'll have to ask LlamaLad about if this could be from the Mixin Extras update. Thanks, so much!

**Comment by BlueCorallite** [comment]
No problem, thank you

**Comment by chronosacaria** [comment]
@BlueCorallite Hey, mate, could you create a Mixin Dump for me so that I can see what is going on? If you are using the vanilla launcher, you can do this by going to your instance and clicking edit. Then, at the bottom, in the `JVM ARGUMENTS` section, you can add this to the end: `-Dmixin.debug.export=true`. This will create a dump when it crashes in the `.minecraft\.mixin.out` directory. If you could zip that up and post it here, that would be awesome! There is no personally identifiable information in those files, so you're safe to do so. I've added a picture so you can see what it looks like in the Edit Installation screen. The example I have is for 1.19.4, but it is the same for any other version
![image](https://github.com/chronosacaria/MCDungeonsWeapons/assets/67794130/e5934b10-b8d8-46b6-9dfb-6df38fdedf5c)

**Comment by BlueCorallite** [comment]
Will do

**Comment by BlueCorallite** [comment]
[class.zip](https://github.com/chronosacaria/MCDungeonsWeapons/files/11492884/class.zip)
Is this it?

**Comment by chronosacaria** [comment]
@BlueCorallite Yep, that's perfect! Thank you!

**Comment by chronosacaria** [comment]
@BlueCorallite Could you try removing MemoryLeakFix and see if that resolves the issue?

**Comment by BlueCorallite** [comment]
Okay, I'll try

**Comment by BlueCorallite** [comment]
Huh. It works.

**Comment by BlueCorallite** [comment]
Tbh I could probably do without the MemoryLeakFix mod, but is there anyway I can still use it?

**Comment by chronosacaria** [comment]
Awesome! I'll report back. I'm definitely not going to consider this resolved because that means that there is some kind of conflict with a very, very important mod. Thanks for all of your testing!

**Comment by chronosacaria** [comment]
There is something happening with MemoryLeakFix and MCDW, which I think I know what the issue is. I'm gonna ask some of my other dev friends and see what they think

**Comment by BlueCorallite** [comment]
No problem, hope it gets fixed soon.
Thank you for your help as well.
All mod creators have my respect and appreciation.

**Comment by chronosacaria** [comment]
@BlueCorallite Okay, so here is something that it appears that you can do in the meantime:

Replace MemoryLeakFix version 1.0.0 with version 0.8.0. You can find it either on [Modrinth](https://modrinth.com/mod/memoryleakfix/version/v0.8.0-1.19.x) or [CurseForge](https://legacy.curseforge.com/minecraft/mc-mods/memoryleakfix/files/4428728). That seems to launch okay

**Comment by BlueCorallite** [comment]
Okay sure, I'll try it out

**Comment by BlueCorallite** [comment]
yeah, 0.8.0 seems to work. Thanks!

**Comment by chronosacaria** [comment]
No problem!
